modules/aws/vpc/vpc-subnet-cni.tf

```diff
@@ -0,0 +1,56 @@
+resource "aws_vpc_ipv4_cidr_block_association" "cni" {
+  vpc_id     = aws_vpc.cluster_vpc.id
+  cidr_block = var.aws_cni_cidr_block
+}
+
```

Comment:
using VPC cird clock association to be as close to TC as possible ( also  increasing VPC size is not really possible so this could be a problem for already existing installations)